README.md

```diff
@@ -26,12 +28,14 @@ In this project, an application was created where the user could make a RomCom c
  - Developing a good understanding of how query selectors and event handlers work together
  - Creating functionality when the user clicks on a button
  - How to properly use the property of classList by adding or removing the class of hidden to certain elements
+ - Used event.preventDefault() to prevent the homepage loading without using the input values for the displayed cover when clicking 'Make my book' button
```

Comment:
Love love love that we removed the preventDefault() to see what would happen and got our answer of why we really needed it!